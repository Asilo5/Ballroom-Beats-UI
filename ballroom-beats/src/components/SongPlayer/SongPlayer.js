import React from 'react';
import { StyleSheet, Text, View } from 'react-native';

export const SongPlayer = () => {
  return (
    <View style={styles.container}>
      <Text>Song Player</Text>
    </View>
  ); 
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: 'white',
    alignItems: 'center',
    justifyContent: 'center',
  },
});

export default SongPlayer;