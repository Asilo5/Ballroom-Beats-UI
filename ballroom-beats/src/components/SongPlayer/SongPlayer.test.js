import React from 'react';
import renderer from 'react-test-renderer';
import SongPlayer from '../SongPlayer/SongPlayer';

describe('SongPlayer', () => {

   it('should match snapshot', () => {
    const player = renderer.create(<SongPlayer />).toJSON();
    expect(player).toMatchSnapshot();
   });
 
});