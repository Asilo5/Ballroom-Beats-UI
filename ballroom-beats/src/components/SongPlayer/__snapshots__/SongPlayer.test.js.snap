// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SongPlayer should match snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Song Player
  </Text>
</View>
`;
